11_notify/n0npax/pkg/puppy/rest.go

```diff
@@ -0,0 +1,132 @@
+package puppy
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+
+	middleware "github.com/anz-bank/go-course/11_notify/n0npax/pkg/middleware"
+	"github.com/gin-gonic/gin"
+)
+
+var (
+	LostPuppyURL        = ""
+	SlowRequestDuration = "2s"
+	PuppyDeleteNotifyF  = LostPuppyReq
```

Comment:
Why do you make this overridable?
Please add at least a comment.
Would be better if you found a way of just using LostPuppyReq (try httptest.Server & channels).

Reply:
fixed!